src/uct/ib/base/ib_md.c

```diff
@@ -282,6 +282,7 @@ static ucs_status_t uct_ib_md_query(uct_md_h uct_md, uct_md_attr_v2_t *md_attr)
     md_attr->access_mem_types          = UCS_BIT(UCS_MEMORY_TYPE_HOST);
     md_attr->detect_mem_types          = 0;
     md_attr->dmabuf_mem_types          = 0;
+    md_attr->reg_nonblock_mem_types    = UCS_BIT(UCS_MEMORY_TYPE_HOST);
```

Comment:
1. it's not really supported on devx
2. what about other md's? any md that has dummy registration or registration without pinning should support it.